scripts/helpers/plugin_functions.sh

```diff
@@ -82,10 +82,11 @@ tpm_plugins_list_helper() {
 # 2. "user/plugin_name"
 plugin_name_helper() {
 	local plugin="$1"
-	# get only the part after the last slash, e.g. "plugin_name.git"
-	local plugin_basename="$(basename "$plugin")"
+	# get only the last part
```

Comment:
nit: This comment provides less context than the original, I think remove it, since the comment for the function is doing the same job.